config/environments/development.rb

```diff
@@ -33,8 +33,21 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```

Comment:
take host from an ENV, defaulting to `localhost:3000` .. `ENV.fetch('APPLICATION_URL', 'localhost:3000')`